mirrord/cli/src/container/command_builder.rs

```diff
@@ -127,6 +127,19 @@ impl RuntimeCommandBuilder {
             extra_args,
         }
     }
+
+    pub fn into_extension_result(self) -> RuntimeCommandExtensionParams {
```

Comment:
I don't get the function name - what is the result?

And why not implement `From<RuntimeCommandBuilder> for RuntimeCommandExtensionParams`?

Reply:
I changed the name